docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/normalize/index.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: "Normalize"
+sidebar_position: 500
+hide_table_of_contents: true
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import { DbtCongfigurationPage } from '@site/src/components/JsonSchemaValidator';
+import { dump } from 'js-yaml';
+import ReactMarkdown from 'react-markdown';
+import Admonition from '@theme/Admonition';
+import DbtSchemas from "@site/docs/reusable/dbt-schemas/_index.md";
+import CodeBlock from '@theme/CodeBlock';
+import { SchemaSetterWSeeds } from '@site/src/components/DbtSchemaSelector';
+
+export const printSchemaVariables = (manifestSchema, scratchSchema, derivedSchema, seedSchema) => {
+  return(
+    <>
+    <CodeBlock language="yaml">
+    {`models:
+  snowplow_normalize:
+    base:
+      manifest:
+        +schema: ${manifestSchema}
+      scratch:
+        +schema: ${scratchSchema}`}
+        </CodeBlock>
+    </>
+  )
+}
+
+export const printYamlVariables = (data) => {
+  return(
+    <>
+      <h4>Project Variables:</h4>
+      <CodeBlock language="yaml">{dump({vars: {"snowplow_normalize": data}}, { flowLevel: 3 })}</CodeBlock>
+    </>
+  )
+}
+
+export const MyMdxComponent = () => {
+  const schemaName = 'dbtNormalize';
+  const versions = ['0.3.3'];
+  const label = 'dbt normalize version';
+  const output = printYamlVariables;
+  const group = 'true';
+  const dbtConfigOutput = DbtCongfigurationPage({
+    schemaName,
+    versions,
+    label,
+    output,
+    group
+  });
+  if (dbtConfigOutput[0]) {
+    return (
+      <div>
+        {dbtConfigOutput[1]} {/* SelectSchemaVersion */}
+        <br/>
+        <h2 id="Variables">Package Configuration Variables</h2>
+        <p>This package utilizes a set of variables that are configured to recommended values for optimal performance of the models. Depending on your use case, you might want to override these values by adding to your <code>dbt_project.yml</code> file.</p>
+        <Admonition type="note">
+          All variables in Snowplow packages start with `snowplow__` but we have removed these in the below tables for brevity.
+        </Admonition>
+        <br/>
+        {dbtConfigOutput[2]} {/* JsonToTable */}
+        <br/>
+        <Admonition type="tip">
+          When modifying the <code>session/user_identifiers</code> or using <code>session/user_sql</code> in the normalize package these will overwrite the <code>domain_sessionid</code> and <code>domain_userid</code> fields in tables, rather than being <code>session/user_identifier</code> as in the core utils implementation. This is for historic reasons to mitigate breaking changes. Original values for these fields can be found in `original_domain_session/userid` in each table.
```

Comment:
This stuff doesn't exist in normalize (or most packages other than web for this version actually) so we should remove it until the base package is added in.